myapp/app/views/layouts/application.html.erb

```diff
@@ -7,9 +7,18 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <!-- Bootstrap CSS -->
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Why 4.5.2? If you want to use a latest version of 4.x.x then you should use v4.6.2.
Or, you want to use a latest version of bootstrap then you should use v5.3.2.
https://www.bootstrapcdn.com/

Reply:
Thank you. I've made it the latest version！